application-analytics-default/src/main/java/com/xwiki/analytics/script/AnalyticsScriptService.java

```diff
@@ -66,4 +70,15 @@ public JsonNode getMatomoRequestResult(Map<String, String> parameters, Map<Strin
             throw new RuntimeException(String.format("Failed to get data for [%s]", jsonNormaliserHint), e);
         }
     }
+
+    /**
+     *
+     * @return the analytics configuration.
+     * @since 1.0
```

Comment:
This should also be removed, as I mentioned in https://github.com/xwikisas/application-analytics/pull/39/files#r1374161258